src/messages/MMDSFindIno.h

```diff
@@ -19,8 +19,8 @@
 #include "include/filepath.h"
 
 struct MMDSFindIno : public Message {
-  ceph_tid_t tid;
-  inodeno_t ino;
+  ceph_tid_t tid = 0;
+  inodeno_t ino = {0};
 
```

Comment:
nit: There is no need of `=` if you do it this way `{0}`. Change both lines to same method of initialization.